sphinx4-core/src/main/java/edu/cmu/sphinx/util/props/ConfigurationManagerUtils.java

```diff
@@ -660,10 +663,10 @@ public static void setProperty(ConfigurationManager cm, String componentName, St
                 ps.setBoolean(propName, Boolean.valueOf(propValue));
                 break;
             case DOUBLE:
-                ps.setDouble(propName, new Double(propValue));
```

Comment:
Can you explain this change? `new Double` creates an object and `Double#parseDouble()` a primitive which is preferable.